calebwagner/uteach-ilearn-server
# adding image url field to serializers

#### Changes Made
1. Modified file `meetings.py` to include `image_url` in AppUser serializer.
​1. Modified file `users.py` to include `image_url` in AppUser serializer.

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout cjw-styling-app
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > Confirm file modifications are present as indicated above.
